cinema/serializers.py

```diff
@@ -114,10 +117,10 @@ class TicketSerializer(serializers.ModelSerializer):
     def validate(self, attrs):
         data = super(TicketSerializer, self).validate(attrs=attrs)
         Ticket.validate_ticket(
-            attrs["row"], 
-            attrs["seat"], 
-            attrs["movie_session"].cinema_hall, 
-            ValidationError
+            attrs["row"],
+            attrs["seat"],
+            attrs["movie_session"].cinema_hall,
+            ValidationError,
```

Comment:
Ensure that the `attrs` dictionary contains the keys 'row', 'seat', and 'movie_session' before accessing them. This is important to avoid potential `KeyError` exceptions if any of these keys are missing.